doc/api/fs.markdown

```diff
@@ -832,6 +836,8 @@ start at 0. The `encoding` can be any one of those accepted by [Buffer][].
 
 If `fd` is specified, `ReadStream` will ignore the `path` argument and will use
 the specified file descriptor. This means that no `open` event will be emitted.
+Note that `fd` should be blocking; non-blocking `fd`s could be passed to
+`net.Socket`.
```

Comment:
`fcntl` can set blocking mode after open, so imo "opened in blocking mode" may be inaccurate.

Okay will s/could/should/, not being a native speaker I have to trust you :)